### PR TITLE
ci: Prepare for the 0.10.0 release

### PR DESCRIPTION
**Pull request by wai-wong-edb**

### CODE REVIEW

**Comment by edbamolvyavahare** [comment]
LGTM

**Review by edbamolvyavahare — CHANGES_REQUESTED** [review]
Did we  add postGIS support to faraway replica resource ?

**Comment by wai-wong-edb** [comment]
> Did we add postGIS support to faraway replica resource ?

No not yet

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+## v0.10.0 (April 25. 2024)
```
Comment (edbamolvyavahare) [inline CHANGELOG.md:1]: need to update date

**Review by edbamolvyavahare — DISMISSED** [review]
LGTM

**Review by edbamolvyavahare — APPROVED** [review]
LGTM

**Pull request merged**